components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
+  }
+
+  // Handle "Enter" key to open/close collapsible
+  function handleKeyDown(e) {
+    if (e.keyCode === 13) handleActive();
```

Comment:
`e.key === "Enter"` could make this a bit more readable

Reply:
I agree. I was unsure what the best practice was.